lib/sidekiq/job_retry.rb

```diff
@@ -134,6 +137,15 @@ def local(jobinst, jobstr, queue)
 
     private
 
+    def process_requeue(jobinst, jobstr)
+      retry_backoff = jobinst.class.get_sidekiq_options.dig("iteration", "retry_backoff") ||
```

Comment:
Remember that sidekiq_options get persisted into the job payload when the job is created; you don't need to dig the value out of `get_sidekiq_options`, you'd use `job.dig("iteration", "retry_backoff")`.

Reply:
Oh, I see; you didn't add these to the Job default options. That makes sense if overriding these elements is rare or you want to keep the job payload size as small as possible. If the elements are overridden, the job payload will contain the values so you can do:

```ruby
job.dig(...) || lifecycle.config.dig(...) # get from specific job or capsule's config
```